src/modules/accommodations/dto/request/create-accommodation.dto.ts

```diff
@@ -23,11 +24,18 @@ export class CreateAccommodationDto {
   @ApiProperty({ default: 'A beautiful villa near the beach.', maxLength: 255 })
   description: string;
 
-  @IsString()
   @IsNotEmpty()
-  @MaxLength(200)
-  @ApiProperty({ default: 'Malibu, California', maxLength: 200 })
-  location: string;
+  @IsObject()
+  @TransformToJson()
```

Comment:
Is it required to parse JSON? All request bodies should be parsed from JSON to JS objects automatically in nest

Reply:
if you have latitude and longitude as separate fields instead of putting them in object, you can have better parsing and it will be easier to handle